cmd/eksctl-anywhere/cmd/flags.go

```diff
@@ -0,0 +1,38 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+)
+
+func addClusterFlags(cc *cobra.Command, clusterOpt *clusterOptions) {
+	cc.Flags().StringVarP(&clusterOpt.fileName, "filename", "f", "", "Filename that contains EKS-A cluster configuration")
+
+	cc.Flags().StringVar(&clusterOpt.bundlesOverride, "bundles-override", "", "Override default Bundles manifest (not recommended)")
+
+	cc.Flags().StringVar(&clusterOpt.managementKubeconfig, "kubeconfig", "", "Management cluster kubeconfig file")
+
+	cc.Flags().BoolVar(&clusterOpt.forceCleanup, "force-cleanup", false, "Force deletion of previously created bootstrap cluster")
+}
+
+func overrideFlagNameAndUsage(cc *cobra.Command, flagName string, newName string, shorthand string, usage string) {
+	f := cc.Flags().Lookup(flagName)
+	if f != nil {
+		f.Name = newName
+		f.Shorthand = shorthand
+		f.Usage = usage
+	}
+}
+
+func bindFlagsToViper(cmd *cobra.Command, args []string) error {
+	var err error
+
```

Comment:
Whitespace

Reply:
Removed whitespace in bindFlagsToViper func